admin/app/components/solidus_admin/ui/forms/search/component.html.erb

```diff
@@ -23,8 +23,7 @@
       ) %>
     </div>
 
-    <details class="px-6 relative overflow-visible">
-      <summary class="hidden"></summary>
```

Comment:
I think the reason for complaint is this `hidden` class, no? So, I agree that a `details` element with a visually hidden `summary` element does not make any sense.

Reply:
We could make it accessible by adding an actual `summary` ("show/hide search results").